src/Tizen.NUI/src/public/WebView/WebView.cs

```diff
@@ -554,7 +542,7 @@ public int DefaultFontSize
         /// Loads a html.
         /// <param name="url">The path of Web</param>
         /// </summary>
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 8 </since_tizen>
         public void LoadUrl(string url)
```

Comment:
Can you please make it hidden again? We are planing to fix XXXUrl api to use System.Uri type.

Reply:
done.